tests/integration/fixtures.py

```diff
@@ -112,6 +112,30 @@ def datetime_table():
     )
 
 
+@pytest.fixture()
+def parquet_ddf(tmpdir):
+
+    # Write simple parquet dataset
+    df = pd.DataFrame(
+        {
+            "a": [1, 2, 3] * 5,
+            "b": range(15),
+            "c": ["A"] * 15,
+            "d": [
+                pd.Timestamp("2013-08-01 23:00:00"),
+                pd.Timestamp("2014-09-01 23:00:00"),
+                pd.Timestamp("2015-10-01 23:00:00"),
+            ]
+            * 5,
+            "index": range(15),
+        },
+    )
+    dd.from_pandas(df, npartitions=3).to_parquet(os.path.join(tmpdir, "parquet"))
```

Comment:
Not exactly sure why, but looks like this `from_pandas` call is resulting in killed workers in the independent cluster testing 😕 I can look into this locally and follow up with more info

Reply:
Would it help to use pandas directly for the write here?

Reply:
No we should be good with this method - the failures would likely arise with other fixtures anyways. Hopefully this should get resolved with https://github.com/dask-contrib/dask-sql/pull/437

Reply:
Okay, sounds good. I'll assume this is good to go then - Let me know if I should address anything else :)